commands.js

```diff
@@ -610,7 +662,7 @@ var commands = exports.commands = {
 		}
 
 		if (targetUser) targetUser.updateIdentity(room.id);
-		if (room.chatRoomData) Rooms.global.writeChatRoomData();
+		if (room.chatRoomData && !room.isPersonal) Rooms.global.writeChatRoomData();
```

Comment:
``chatRoomData`` is created on the ``ChatRoom`` constructor, but yeah, can be changed so it doesn't if necessary.